examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -145,12 +145,10 @@ class CastingPlayer : public std::enable_shared_from_this<CastingPlayer>
      * CommissionableDataProvider
      * (matter::casting::core::CastingApp::GetInstance()->UpdateCommissionableDataProvider(CommissionableDataProvider)).
      *
-     * @param connectionCallbacks contains the ConnectCallback and CommissionerDeclarationCallback (Optional).
-     * @param commissioningWindowTimeoutSec (Optional) time (in sec) to keep the commissioning window open, if commissioning is
-     * required. Needs to be >= kCommissioningWindowTimeoutSec.
+     * The same connectionCallbacks and commissioningWindowTimeoutSec parameters passed in to VerifyOrEstablishConnection() will be
```

Comment:
In the docs above for VerifyOrEstablishConnection, can you add a mention of how ContinueConnecting is related to the former i.e. in what cases / on what callbacks may ContinueConnecting need to be called?

Reply:
Yes.